hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/ChunkInputStream.java

```diff
@@ -628,12 +645,38 @@ private boolean chunkStreamEOF() {
     }
   }
 
+
+  /**
+   * Release a buffers upto the given index.
```

Comment:
Tiny nit: make sure to change it before commit
Release a buffers --> Release the buffers